### ISSUE TITLE
Colours for parties

### ISSUE THREAD

**Issue by AlanI-xx** [belowtheline/site_2014#19]
This is a part suggestion, part question: 

The big parties have their name and code are all specially coloured when they appear on the site, specifically: ALP, GRN, LIB/LNP/CLP, NAT

Are there any thoughts on assigning colours to some of the larger remaining parties, like KAP, PUP, FF?

I'm somewhat using the ABC's colouring as a guide: http://www.abc.net.au/news/federal-election-2013/guide/candindex/

I suppose in the grand scheme of things it doesn't matter -- but it does look a little nicer.

**Comment by jeamland** [comment]
Yeah, I considered this. The problem is coming up with the actual colour values. The ones I've got were stolen from Bootstrap who happened to conveniently have the colours I needed. =)

If you can come up with colours for some of the other "big name" parties that fit in then I'm happy to look at putting them in.
